gpjax/variational_families.py

```diff
@@ -45,21 +45,27 @@ def predict(self, *args: Any, **kwargs: Any) -> dx.Distribution:
 
 
 @dataclass
-class VariationalGaussian(AbstractVariationalFamily):
+class AbstractVariationalGaussian(AbstractVariationalFamily):
+    """The variational Gaussian family of probability distributions."""
+
+    prior: Prior
+    inducing_inputs: f64["N D"]
+    name: str = "Gaussian"
+    jitter: Optional[float] = DEFAULT_JITTER
+
+
+@dataclass
+class VariationalGaussian(AbstractVariationalGaussian):
     """The variational Gaussian family of probability distributions.
 
     The variational family is q(f(·)) = ∫ p(f(·)|u) q(u) du, where u = f(z) are the function values at the inducing inputs z
     and the distribution over the inducing inputs is q(u) = N(μ, S). We parameterise this over μ and sqrt with S = sqrt sqrtᵀ.
 
     """
 
-    prior: Prior
-    inducing_inputs: f64["M D"]
-    name: str = "Variational Gaussian"
-    variational_mean: Optional[f64["M Q"]] = None
-    variational_root_covariance: Optional[f64["M M"]] = None
+    variational_mean: Optional[f64["N D"]] = None
+    variational_root_covariance: Optional[f64["N D"]] = None
```

Comment:
Not `f64["N N"]`?

Reply:
Variational families are revised in d84ede9.